ecs-agent/manageddaemon/managed_daemon.go

```diff
@@ -33,6 +33,12 @@ const (
 	defaultApplicationLogMount            = "applicationLogMount"
 )
 
+var (
```

Comment:
We shouldn't hard code this -- the daemons should be dynamic.  There may or may not be any number of managed daemons.  
A thought: can you wire these in from the managed daemon ImportAll path?  
every time the agent starts up it will create an array of managed daemons.  we can use the names of these daemons as our source list for checking against state.

Reply:
per offline, we'll always take the image name at face value (i.e. no need to have a hardcoded list of managed daemon images)